src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1331,12 +1332,13 @@ class CreateVmModal extends React.Component {
 
         const detailsTab = (
             <>
+                {superuser.allowed &&
```

Comment:
Hmm, I think people can create polkit rules to allow non-priv users access to the "system" connection. In fact cockpit-machines always connects to both "system" and "session" on initialization and tolerates AccessDenied errors for non-priv users.

So I think we should make a new global flag that says "can't access system connection", and use that instead of `superuser.allowed` in this PR.  The place to set that flag is app.jsx:94, I guess.

Reply:
Polkit is a disaster if you ask me, we have no way to detect it. But plumbing the state through should work. But looking at our code this is all a bit confusing I guess we look at `systemSocketInactive`?

Reply:
This turns into a yakshave...

```
    } else if ((superuser.allowed && systemSocketInactive) || loadingResources) {
```

So seems my solution is "correct" :) But ofcourse we can rework this.

Reply:
Or not, not sure why this even exists.